src/utils/schema/resolvers/Query/all.ts

```diff
@@ -703,7 +735,7 @@ export default (entityName: string, fieldsByName: any, idsByName: any) =>
 
       // === CONSTRUCT OUTPUT + RETURN ===
       const edges = items.map((r) => {
-        const record = getAccessibleFields(r, ability);
+        const record = getAccessibleFields(r, abilityCache.get(userId));
```

Comment:
@SianaDicheva 
it's really an edge case but there is a possibility there that the cache is cleared, between the time you add the user ability to the cache, and the time you get the accessible fields
as the cache is automatically cleared every 10minutes, regardless of when you added a new user or form to it, this situation may happen

Reply:
@SianaDicheva 
I'll make some changes to the PR, please don't push any new code